media/css/eoy.css

```
@@ -94,152 +220,207 @@ body {
   line-height: 1.6;
 }
 
-@media (prefers-color-scheme: dark) {
-  body {
-    background-color: #204571;
-    background-image: url('/media/img/thunderbird/backgrounds/forest-bg-dark.jpg');
-  }
+#initAppeal {
+  display: flex;
+  gap: 9px;
+  margin: 0 auto;
+  margin-bottom: 100px;
+  padding: 24px;
+  max-width: 1000px;
+  align-items: center;
+  font-size: 1rem;
 }
 
-header {
-  margin: 0 auto;
-  max-width: 200ch;
+#initAppeal p strong {
+  font-size: 1.1rem;
 }
 
-#initAppeal {
-  display: grid;
-  grid-template-columns: 1fr auto;
-  gap: 2em;
-  padding: 1em;
-  margin-bottom: 4em;
-  align-items: center;
+#initAppeal h1 {
+  font-size: 5rem;
+  line-height: 1;
 }
 
-@media (max-width: 800px) {
+@media (max-width: 720px) {
   #initAppeal {
-    grid-template-columns: 1fr;
+    flex-direction: column-reverse;
     justify-items: center;
   }
 }
 
 main {
   padding: 1em;
   margin: 0 auto;
-  max-width: 100ch;
+  max-width: 800px;
+  position: relative;
+}
+
+#bg-gradient {
+  position: absolute;
+  inset: 0;
+  top: 500px;
+  background-image: linear-gradient(rgba(76,177,249,0.2), var(--color-blue-40));
+  z-index: -1;
+}
+@media (prefers-color-scheme: dark) {
+  #bg-gradient {
+    background-image: linear-gradient(rgba(19,115,217,0.2), var(--color-blue-90));
+  }
 }
 
 .accent-text {
   color: var(--accent-text);
 }
 
 #illustration > svg {
-  max-width: 400px;
-  max-height: 400px;
+  max-width: 300px;
+  height: auto;
 }
 
 .donate-banner {
-  --accent-text: #7F1D1D;
+  --accent-text: var(--color-red-80);
 
-  background: rgb(255,59,59);
-  background-image: url('/media/img/thunderbird/backgrounds/hearts-bg.png'),
-                    linear-gradient(125deg, rgba(255,59,59,1) 0%, rgba(168,33,33,1) 45%, rgba(255,230,230,1) 45%, rgba(255,241,241,1) 100%);
+  background: var(--color-red-50);
+  background-image: linear-gradient(125deg, var(--color-red-50) 0%, var(--color-red-70) 45%, var(--color-red-10) 45%, var(--color-gray-10) 100%);
   background-position: top left, center;
   background-size: contain;
   background-repeat: no-repeat;
   border-radius: 3px;
+  box-shadow: 4px 4px 24px rgba(0,0,0,0.15);
   color: white;
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  gap: 1em;
+  display: flex;
+  justify-content: space-between;
   font-size: 1.25rem;
-  padding: 1em;
-  margin: 2em;
-  margin-top: -75px;
+  padding: 12px;
+  margin: 0 auto;
+  margin-top: -114px;
   text-decoration: none;
-  box-shadow: var(--shadow);
+  text-shadow: var(--shadow);
   transition: transform 0.3s ease-in-out;
   position: relative;
   overflow: hidden;
+  width: 400px;
+  height: 100px;
```

Without the padding, we can make this button the size we want and we can be sure it won't change.
```suggestion
  width: 430px;
  height: 120px;
```